authexternalbrowser_test.go

```diff
@@ -112,3 +113,23 @@ func TestUnitAuthenticateByExternalBrowser(t *testing.T) {
 		t.Fatalf("unexpected error code. expected: %v, got: %v", ErrCodeFailedToConnect, driverErr.Number)
 	}
 }
+
+func TestAuthenticationTimeout(t *testing.T) {
+	authenticator := "externalbrowser"
+	application := "testapp"
+	account := "testaccount"
+	user := "u"
+	password := "p"
+	timeout := 0 * time.Second
```

Comment:
Am I right, that this test ends immediately?

Reply:
Yes, it does